pkg/config/load.go

```diff
@@ -33,6 +33,21 @@ type Prowgen struct {
 	Expose bool `json:"expose,omitempty"`
 	// Rehearsals declares any disabled rehearsals for jobs
 	Rehearsals Rehearsals `json:"rehearsals,omitempty"`
+	// Set which cluster and architecture should the images be promoted from
+	Images Images `json:"postsubmit_images"`
+}
+
+type Images struct {
+	// amd64 is the default arch, falsify this field to skip it
+	SkipDefaultArch bool   `json:"skip_default_arch,omitempty"`
+	Architectures   []Arch `json:"architectures"`
+}
+type Arch struct {
+	// The name of the architecture, it will be used as suffix for the job
+	// Ex.: openshift-ci-tools-master-images-${arch.Name} -> openshift-ci-tools-master-images-arm64
+	Name string `json:"name,omitempty"`
```

Comment:
This is not needed actually. You can get the name from the name of the architecture that we will hardcode.

Reply:
See the answer above.